src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteJenkinsServer.java

```diff
@@ -70,12 +80,32 @@ public String getDisplayName() {
         return displayName;
     }
 
-    public URL getAddress() {
-        return address;
+    public boolean getHasBuildTokenRootSupport() {
+        return hasBuildTokenRootSupport;
+    }
+    
+    public Auth2 getAuth2() {
```

Comment:
Should this just `readObject` into a `Auth2` instance when loaded from disk, and done?